packages/suite/src/components/suite/FiatValue.tsx

```diff
@@ -15,12 +15,6 @@ import { HiddenPlaceholder } from 'src/components/suite';
 
 import { HiddenPlaceholderProps } from './HiddenPlaceholder';
 
-const StyledHiddenPlaceholder = styled((props: HiddenPlaceholderProps) => (
-    <HiddenPlaceholder {...props} />
-))`
-    font-variant-numeric: tabular-nums;
```

Comment:
this is already [on HiddenPlaceholder always](https://github.com/trezor/trezor-suite/blob/e22b683733bf9069ac9a7e0190d63b35697a954f/packages/suite/src/components/suite/HiddenPlaceholder.tsx#L17)